### PR TITLE
Ft dashboard

### LINKED ISSUE #5: dashboard layout @daniel
dashboard layout  with dynamic data and other temporary placeholders @daniel e.g biodata, etc

### PR DESCRIPTION
**Pull request by danielufeli**
## Description
Please include a summary of the change and relevant motivation and context. 
- Developed Dashboard Page
- Added Routing using react-router-dom

List any dependencies that are required for this change.
- React Route Dom

All pull requests should be related to open issues. Indicate the issue(s) bellow and remove this line:
Fixes #(5)

## How Has This Been Tested?
http://localhost:3000/dashboard/

## Screenshots (if applicable, else remove this line / section)
image

## Checklist:

- [x ] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [x ] I have added necessary inline code documentation
- [x ] I have added tests that prove my fix is effective and that this feature works
- [x ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by nextwebb — DISMISSED** [review]

File: .cache/1f/d9e20c8dddfb418f6a796f5b245774.json [hunk 1]
```diff
@@ -0,0 +1 @@
...y(exports, \"__esModule\", {\n  value: true\n});\nexports.default = void 0;\n\nconst app = async () => '#BuildforSDG';\n\nvar _default = app;\nexports.default = _default;"},"sourceMaps":{"js":{"mappings":[{"generated":{"line":8,"column":0},"source":"LandingPage/landingPage.js","original":{"line":1,"column":0}},{"name":"app","generated":{"line":8,"column":6},"source":"LandingPage/landingPage.js","original":{"line":1,"column":6}},{"generated":{"line":8,"column":9},"source":"LandingPage/landingPage.js","original":{"line":1,"column":9}},{"generated":{"line":8,"column":12},"source":"LandingPage/landingPage.js","original":{"line":1,"column":12}},{"generated":{"line":8,"column":24},"source":"LandingPage/landingPage.js","original":{"line":1,"column":24}},{"generated":{"line":8,"column":38},"source":"LandingPage/landingPage.js","original":{"line":1,"column":0}},{"name":"app","generated":{"line":10,"column":15},"source":"LandingPage/landingPage.js","original":{"line":3,"column":15}}],"sources":{"LandingPage/landingPage.js":"const app = async () => '#BuildforSDG';\r\n\r\nexport default app;\r\n"},"lineCount":null}},"error":null,"hash":"240a4e88515f8fd3d414d1d79c899ad9","cacheData":{"env":{}}}
```
Comment (triple0t) [inline .cache/1f/d9e20c8dddfb418f6a796f5b245774.json:1]: Add .cache to your gitignore file

File: package.json [hunk 2]
```diff
@@ -1,49 +1,40 @@
...
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react": "^9.3.2",
+    "@testing-library/user-event": "^7.1.2",
+    "bootstrap": "^4.5.0",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
+    "react-icons-kit": "^1.3.1",
+    "react-router-dom": "^5.2.0",
+    "react-scripts": "3.4.1",
+    "reactstrap": "^8.4.1"
```
Comment (triple0t) [inline package.json:17]: Why are you using so many UI libraries?

File: public/index.html [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    <script
+      src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js"
+      integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo"
+      crossorigin="anonymous"
+    ></script>
+    <script
+      src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js"
+      integrity="sha384-OgVRvuATP1z7JjHLkuOU7Xw704+h835Lr+6QL9UvYjZE3Ipu6Tp75j7Bh/kR0JKI"
+      crossorigin="anonymous"
+    ></script>
```
Comment (triple0t) [inline public/index.html:60]: Did you consider using Bootstrap with support for React e.g [react-bootstrap](https://react-bootstrap.netlify.app/)
I can see you are making use of [reactstrap](https://reactstrap.github.io/)
Adding extra packages you won't use is not very nice

File: src/App.test.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
+  const { getByText } = render(<App />);
+  const linkElement = getByText(/learn react/i);
+  expect(linkElement).toBeInTheDocument();
```
Comment (triple0t) [inline src/App.test.js:8]: You need to update your Test Cases

**Review by triple0t — APPROVED** [review]
I will Approve this but please, kindly update before Merging
Thanks

**Review by nextwebb — APPROVED** [review]
I'll approve but your code failed all the tests :(

**Pull request merged**